### PR TITLE
Run the tests on Android Chrome

### PR DESCRIPTION
**Pull request by mithro**

### CODE REVIEW

**Comment by alancutter** [comment]
Looks like we might need to drop our accuracy to two decimal places...

File: run-tests-android.sh [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  # Install the apk
+  echo "Installing Chrome"
+  $ADB install $CHROME_APK
+  $ADB shell input keyevent 82   # Send the menu key to unlock the screen
+  $ADB shell am start -a android.intent.action.MAIN -n $CHROME_APP/.Main  # Start chrome
+
+  if kill -0 $EMULATOR_PID; then
+    break
+  else
+    echo "Emulator has crashed, try to start it again..."
```
Comment (dstoc) [inline run-tests-android.sh:76]: try -> trying?

File: test/test-runner.html [hunk 2]
```diff
@@ -171,7 +171,11 @@ <h1 id=status>Pending</h1>
  * @return {number} Time as this very moment.
  */
 function now() {
-  return performance.now();
```
Comment (dstoc) [inline test/test-runner.html:174]: this seems unrelated to running the tests on android

File: .travis-setup.sh [hunk 3]
```diff
@@ -7,7 +7,7 @@ ls -l /dev/shm
 sudo chmod 1777 /dev/shm
 ls -l /dev/shm
 
-sudo apt-get update --fix-missing
+sudo apt-get update --fix-missing > /dev/null 2>&1
```
Comment (dstoc) [inline .travis-setup.sh:14]: -qq ?
